src/dtlssrtptransport.cpp

```diff
@@ -84,9 +84,10 @@ bool DtlsSrtpTransport::sendMedia(message_ptr message) {
 
 	int size = message->size();
 	PLOG_VERBOSE << "Send size=" << size;
+//    return outgoing(message);
 
 	// The RTP header has a minimum size of 12 bytes
-	if (size < 12)
+	if (size < 8)
```

Comment:
Out of curiosity, why is this necessary?

Reply:
When I was working on this a couple of days ago, I got RR's from the browser with a report count of zero (thus 8 bytes long). I *think* a receiver/sender report with a report count of zero is legal.

Reply:
I see! Feel free to add this as a comment.